src/ComponentDiscovery.php

```diff
@@ -80,10 +71,10 @@ protected function getBaseComponentPath() {
    */
   public function getAll() {
     if (is_null($this->components)) {
-      $base_path = $this->getBaseComponentPath();
+      $identifier = 'lightning_';
```

Comment:
We use $identifier twice, so it should probably be a class constant. static::COMPONENT_PREFIX or something.